Algorithmia/insights.py

```diff
@@ -0,0 +1,13 @@
+import requests
+import json
+
+
+class Insights:
+    # Example of correct insights:
+    # {
```

Comment:
Would it be possible to use a more streamlined key / value pair structure? For example, what we've written up in [the documentation proposal for Python](https://insights1.enthalpy.click/developers/clients/python#publishing-algorithmia-insights) so far? (Sorry I didn't get this to you last week):
```
client.report_insights({"cats_in_image": 4, "dogs_in_image": 7})
```

Reply:
Got it, I had not seen that document before. I changed the method signatures as well